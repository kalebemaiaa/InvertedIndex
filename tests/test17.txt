QueryOr(QueryWord("embaixador"), QueryAndS(QueryNot(QueryWord("senador")), QueryOr(QueryWord("prefeito"), QueryWord("governador"))))
597.raw
4465.raw
489.raw
4373.raw
11551.raw
11624.raw
11387.raw
2759.raw
2605.raw
5745.raw
2352.raw
3975.raw
8867.raw
255.raw
5444.raw
11745.raw
1310.raw
4309.raw
3610.raw
2814.raw
3544.raw
10968.raw
11004.raw
10958.raw
11691.raw
5134.raw
2308.raw
5812.raw
3349.raw
3689.raw
3225.raw
11782.raw
4338.raw
684.raw
3660.raw
11712.raw
227.raw
881.raw
439.raw
220.raw
11882.raw
2752.raw
4415.raw
5325.raw
6098.raw
2058.raw
4243.raw
2332.raw
3598.raw
5992.raw
3615.raw
3786.raw
3468.raw
5098.raw
4011.raw
2061.raw
3358.raw
59.raw
10791.raw
257.raw
2295.raw
4840.raw
11483.raw
2926.raw
4169.raw
212.raw
576.raw
725.raw
5220.raw
6315.raw
4946.raw
4509.raw
10867.raw
4267.raw
3355.raw
1807.raw
4705.raw
5653.raw
5601.raw
4129.raw
4122.raw
1524.raw
5192.raw
11049.raw
4168.raw
3399.raw
11944.raw
3724.raw
4507.raw
5190.raw
4439.raw
5590.raw
6087.raw
2658.raw
11772.raw
5943.raw
4892.raw
1634.raw
10903.raw
4085.raw
912.raw
5037.raw
11486.raw
125.raw
10808.raw
3106.raw
2164.raw
2942.raw
1180.raw
29.raw
4964.raw
1349.raw
2880.raw
3557.raw
5593.raw
4377.raw
8725.raw
3333.raw
6320.raw
614.raw
5299.raw
5236.raw
1113.raw
5076.raw
1170.raw
3684.raw
1374.raw
3998.raw
2248.raw
4755.raw
3383.raw
3733.raw
2631.raw
2165.raw
976.raw
11964.raw
3245.raw
816.raw
3792.raw
2377.raw
389.raw
2465.raw
4856.raw
5571.raw
6282.raw
5189.raw
4613.raw
581.raw
4985.raw
1799.raw
4366.raw
3426.raw
1261.raw
980.raw
1666.raw
1908.raw
3977.raw
2052.raw
1504.raw
8886.raw
2046.raw
5564.raw
1983.raw
507.raw
4962.raw
5070.raw
3461.raw
219.raw
3394.raw
2424.raw
3995.raw
8827.raw
8833.raw
5749.raw
5194.raw
1866.raw
1974.raw
11574.raw
4091.raw
4861.raw
3866.raw
2654.raw
703.raw
2764.raw
5919.raw
2398.raw
960.raw
5579.raw
5817.raw
11762.raw
1994.raw
10981.raw
1973.raw
965.raw
5139.raw
11253.raw
2406.raw
8753.raw
3116.raw
4708.raw
