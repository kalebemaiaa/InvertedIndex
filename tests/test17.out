212
255
507
576
581
881
912
960
965
976
1666
1799
1807
1983
1994
2052
2058
2332
2352
2406
3225
3333
3383
3557
3615
3689
3724
3792
3977
4085
4168
4243
4267
4366
4377
4439
4465
4705
4861
5037
5189
5192
5590
5601
6087
11551
11624
11745
